tiger/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "tiger"
+version = "0.1.2"
+description = "Tiger hash function"
+authors = ["Ulrik Mikaelsson <ulrik.mikaelsson@gmail.com>", "RustCrypto Developers"]
+license = "Apache-2.0"
```

Comment:
All of our other crates are `MIT OR Apache-2.0`. Is that acceptable?

Reply:
that's a good question.  The original work is released under Apache 2.0.  From reading https://law.stackexchange.com/questions/6081/can-i-bundle-mit-licensed-components-in-a-apache-2-0-licensed-project it seems like we don't need 

> Looks like you need to `rustfmt`

done

Reply:
It would definitely be good to get the original author's blessing to republish it as MIT+Apache2.0

Reply:
I have emailed him.  I started to make it look like it was under both licenses, but removed that.